lib/galaxy/authnz/managers.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, app, oidc_config_file, oidc_backends_config_file):
             file (e.g., oidc_backends_config.xml).
         """
         self.app = app
+        self.allowed_idps = None
```

Comment:
Maybe I'd move this at the start of `_parse_oidc_backends_config()`, but looks good any way.